install/data/defaults.json

```diff
@@ -116,5 +116,6 @@
     "eventLoopLagThreshold": 100,
     "eventLoopInterval": 500,
     "onlineCutoff": 30,
-    "timeagoCutoff": 30
+    "timeagoCutoff": 30,
+    "categoryWatchState": "watching"
```

Comment:
Would the logic be simpler if this was `1` for watching, `0` for not, and `-1` for ignoring?